rust/lance-index/src/vector/pq/distance.rs

```diff
@@ -105,6 +105,9 @@ pub(super) fn compute_pq_distance(
     num_sub_vectors: usize,
     code: &[u8],
 ) -> Vec<f32> {
+    if code.is_empty() {
+        return Vec::new();
+    }
```

Comment:
This change was needed to get the existing test to pass in CI. It does seem suspicious though: why would we get an empty code book in PQ if we have nulls?

Reply:
I think it would be fine to merge this for now, to fix the immediate bug, but to come back to this an investigate later.